test/resources/iframe1.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Shouldn't this go after the license header?

Reply:
Same question for the other files

Reply:
I think it might not be parsed be the browser in this case but I am not sure

Reply:
in any case, I think it's not important for the test, I can remove doctype